binderhub/templates/loading.html

```diff
@@ -1,23 +1,25 @@
 {% extends "page.html" %}
 
-{% block head %}
+{% block meta %}
+{{ super() }}
 <meta id="base-url" data-url="{{base_url}}">
 <meta id="badge-base-url" data-url="{{badge_base_url}}">
-<!-- Other OG information in page.html -->
+{% endblock meta %}
+
+{% block meta_social %}
```

Comment:
should meta_social be inside meta?

Reply:
if we put meta_social into meta here in loading.html, then we would have content of meta_social block 2 times in loading page; content of meta_social block in page.html (because of `{{ super() }}`) and also content of meta_social block in this template. actually that's why I didn't want to make blocks nested in the beginning, it is confusing.

Reply:
ah that's a good point - some of the meta needs to be on all pages, while other meta is page-specific.

I wonder if we could use "if/then" statements to control behavior?